### PR TITLE
Integrated Searchbar in the Portfolio page

### PR DESCRIPTION
**Pull request by azteccoding**
I coded this entire component in 1 hour

### CODE REVIEW

**Review by azteccoding — COMMENTED** [review]
This line is good but it is better if you capitlize Searchbar :)

**Review by migratedigital — COMMENTED** [review]
As stated before you should capitalize Searchbar as it is better

File: searchbar.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+searchbar
```
Comment (azteccoding) [inline searchbar.txt]: Deleted

Comment (migratedigital) [inline searchbar.txt]: So please note this is a comment about the code in searchbar txt file

**Review by migratedigital — CHANGES_REQUESTED** [review]
Apart from that everything is ok

File: searchbar.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+<Searchbar />
```
Comment (migratedigital) [inline searchbar.txt:1]: Este es un comentario para cada línea de código

**Review by migratedigital — APPROVED** [review]
Este es el comentario general de la revisión
Puedo comentar, aprobar o solicitar cambios

**Review by migratedigital — APPROVED** [review]
Ese último cambio fue bueno. Listos para el merge!

**Pull request merged**